schema.graphql

```diff
@@ -7779,13 +7779,6 @@ input AttributeInput @doc(category: "Attributes") {
   """
   value: AssignedAttributeValueInput
 
-  """
-  Names corresponding to the attributeValues associated with the Attribute. When specified, it filters the results to include only records with one of the matching values.
-  
-  Added in Saleor 3.22.
-  """
-  valueNames: [String!]
-
```

Comment:
@witoszekdev are you aware of that? Currently `AttributeQueryVarsBuilder.ts` is failing because it access this field